### PR TITLE
Refactor files to fulfill humanmade/coding-standards

### PR DESCRIPTION
**Pull request by onnimonni**
Now the whole project should be compliant against your coding standards. I needed to add
```
//@codingStandardsIgnoreStart
//@codingStandardsIgnoreEnd
```
Into `bin/cavalcade` to exclude the namespacing rules for that file and I excluded `bootstrap.php` as well.

### CODE REVIEW

File: inc/class-runner.php [hunk 1]
```diff
@@ -257,16 +256,18 @@ protected function run_job( $job ) {
 		$cwd = $this->wp_path;
 		printf( '[%d] Running %s (%s %s)' . PHP_EOL, $job->id, $command, $job->hook, $job->args );
 
-		$spec = array(
+		$spec = [
+			//@codingStandardsIgnoreStart
 			// stdin
 			// 0 => null,
```
Comment (onnimonni) [inline inc/class-runner.php]: @rmccue Why is this commented out?

File: bin/cavalcade [hunk 2]
```diff
@@ -4,7 +4,9 @@
  * Cavalcade
  */
 
+//@codingStandardsIgnoreStart
 namespace HM\Cavalcade\Runner;
+//@codingStandardsIgnoreEnd
```
Comment (rmccue) [inline bin/cavalcade]: Can we ignore just the line with `@codingStandardsIgnoreLine` instead?

File: inc/class-runner.php (hunk 1 above)
Comment (rmccue) [inline inc/class-runner.php]: This is there as a hint that we're intentionally avoiding adding a stdin pipe. Rephrasing the comment to that effect would be better than ignoring I think.

File: bin/cavalcade (hunk 2 above)
Comment (onnimonni) [inline bin/cavalcade]: I did not know about `@codingStandardsIgnoreLine`. Yes it's much better!

**Comment by onnimonni** [comment]
Those parts should be okay now :bowtie:

**Pull request merged**